File: Project2_Java/src/pl/pg/oop/project2/organisms/plants/Grass.java
package pl.pg.oop.project2.organisms.plants;

import pl.pg.oop.project2.organisms.OrganismsEnum;
import pl.pg.oop.project2.organisms.Plant;
import pl.pg.oop.project2.organisms.World;

import java.awt.*;

public class Grass extends Plant {
    public Grass(World world, Point position) {
        super(world, position);
        this.TypeOfOrganism = OrganismsEnum.GRASS;
    }

    @Override
    public String toString() {
        return "Grass";
    }
}

File: Project3_Python/Plants/Grass.py
import World
from Plant import Plant
from Organism import OrganismsEnum


class Grass(Plant):
    def __init__(self, world, position):
        super().__init__(world, position)
        self.type_of_organism = OrganismsEnum.Grass

    def __str__(self):
        return "Grass"
